metrics/method_length.py

```diff
@@ -29,7 +31,12 @@ def _parse(self, ast, file, data):
                 node.end_point[0] - node.start_point[0] + 1
             )  # length is zero indexed - therefore we add 1 at the end
             if length > 25:
-                data.append(["Method Length", file, 1, 1])
+                result.append(
+                    Violation(
+                        "method length",
+                        (str(file), node.start_point[0], node.end_point[0] + 1),
```

Comment:
Just want to confirm this will give a location per offending method? That is what I could infer from the test as well

Reply:
It will report a location (<line of method header>, <last line of method>). The result that is reported back, will contain files and lines to said violating method/s.